pages/linux/ufw.md

```diff
@@ -11,26 +11,26 @@
 
 `ufw disable`
 
-- Show ufw rules along with their numbers:
+- Show ufw rules, along with their numbers:
 
 `ufw status numbered`
 
 - Allow incoming traffic on port 5432 on this host:
 
 `ufw allow {{5432}}`
 
-- Allow traffic from 192.168.0.4 to any address on this host on port 22 using only tcp protocol:
+- Allow traffic from 192.168.0.4 to any address on this host, on port 22, using only the tcp protocol:
 
 `ufw allow from {{192.168.0.4}} to {{any}} port {{22}} proto {{tcp}}`
 
 - Deny traffic on port 80 on this host:
 
 `ufw deny {{80}}`
 
-- Deny traffic to port 22 using only udp protocol:
+- Deny traffic to port 22, using only the UDP protocol:
```

Comment:
I was originally inclined to agree with @agnivade but I can see how that wins on a more abstract, intellectual realm, whereas @ostera's arguments win on a more concrete and pragmatic perspective.

The combined example might actually even convey the grammar better than separate ones (where the reader would have to detect the pattern and build it on their heads), so I'd even approve of adopting that. But I'm ok with just rewording the description if we don't get consensus on the latter.

ps - we're discussing on a subset of the changes rather than on the full PR. Unfortunately github's UI seems to be making that distinction rather subtle lately.